aks/dbTemplate.md

```diff
@@ -0,0 +1,195 @@
+
+{% include sub-template-prerequisites-uami.md %}
+
+{% include sub-template-prerequisites-wls.md %}
+
+If you are using your own datasource driver, make sure the datasource driver is uploaded during the WebLogic Server instance provisioning. 
+
+You can create a WebLogic cluster with datasource driver library with steps:
+
+  - Open [WebLogic on AKS marketplace offer](https://portal.azure.com/?feature.customPortal=false#create/oracle.20210620-wls-on-aks20210620-wls-on-aks)
+
+  - Input values to **Basics** and **Config AKS cluster** blade
+
+  - In the Database blade, select **Yes** to connect to database. For **Choose databse type**, select **Other**, upload datasource driver in **Datasource driver**.
+
+  - Finish other inputs, create **Review + create** to provison a cluster.
+
+You will get a WebLogic cluster with your datasource driver built in the image.
+
+### Database Instance
+
+To apply a database with {{ site.data.var.wlsFullBrandName }},
+you must have an existing database instance to use.
+
+#### Work with built-in datasource driver
+
+The marketplace offer ships with database driver for [Oracle](https://ms.portal.azure.com/#blade/Microsoft_Azure_Marketplace/MarketplaceOffersBlade/selectedMenuItemId/home/searchQuery/oracle%20database), [Azure SQL Server](https://docs.microsoft.com/en-us/azure/azure-sql/) and [Azure Database for PostgreSQL](https://docs.microsoft.com/en-us/azure/azure-sql/database/single-database-create-quickstart?WT.mc_id=gallery&tabs=azure-portal).  You can invoke the dbTemplate to create datasource connection for those database. 
+If you do not have an instance, please create one from Azure portal.
+
+#### Bring your own datasource driver
+
+Besides [Oracle](https://ms.portal.azure.com/#blade/Microsoft_Azure_Marketplace/MarketplaceOffersBlade/selectedMenuItemId/home/searchQuery/oracle%20database), [Azure SQL Server](https://docs.microsoft.com/en-us/azure/azure-sql/) and [Azure Database for PostgreSQL](https://docs.microsoft.com/en-us/azure/azure-sql/database/single-database-create-quickstart?WT.mc_id=gallery&tabs=azure-portal), you are able to create datasource connection using other databases, such as [IBM Informix](https://www.ibm.com/products/informix) and [MariaDB](https://mariadb.org/), but you have to follow those steps to achieve that:
+
+  - Create your database server, and make sure the database is accessible from Azure.
+
+  - Ship your database driver to WebLogic cluster. The only approach to upload a datasource driver is using [marketplace offer](https://portal.azure.com/?feature.customPortal=false#create/oracle.20210620-wls-on-aks20210620-wls-on-aks). The offer enables you to bring your own datasource driver. See [WebLogic Server Instance](#webLogic-server-instance)
+
+  - Invoke the dbTemplate to update an existing datasource connection or create another new datasource connection.
+
+### Apply multiple datasource
+
+You may want to enable multiple datasource in your cluster for the following usage:
```

Comment:
enable multiple datasource -> enable multiple data sources

Reply:
Fixed in https://github.com/oracle/weblogic-azure/pull/88/commits/09c58c430464aa1f8ba8a8f2e0efc00dc141e6fb